src/calculateRentalCost.js

```diff
@@ -3,8 +3,18 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
```

Comment:
The name of the variable `three` is not descriptive enough. It's better to use a name that explains its purpose in the code. For example, `SHORT_TERM_RENTAL` would be a more descriptive name.